src/Asset/Widget/Queue.php

```diff
@@ -99,7 +99,7 @@ public function process(Request $request, Response $response)
     {
         /** @var WidgetAssetInterface $widget */
         foreach ($this->queue as $widget) {
-            if ($widget->getZone() === Zone::FRONTEND && $widget->isDeferred()) {
+            if ($widget->isDeferred()) {
                 $this->addDeferredJavaScript($widget, $response);
```

Comment:
Sanity check only… but there was a reason you didn't want deferred widgets in the backend, can you remember what that was?